pkgs/by-name/as/asahi-bless/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, fetchCrate
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "asahi-bless";
+  version = "0.3.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-pgl424SqeHODsjGwNRJtVHT6sKRlYxlXl3esEKK02jc=";
+  };
+
+  cargoHash = "sha256-ilblP8nqb/eY0+9Iua298M7NQKv+IwtdliGd9ZYAVaM=";
+  cargoDepsName = pname;
+
+  meta = with lib; {
+    description = "A tool to select active boot partition on ARM Macs";
+    homepage = "https://crates.io/crates/asahi-bless";
+    license = licenses.mit;
+    maintainers = with maintainers; [ lukaslihotzki ];
+    mainProgram = "asahi-bless";
```

Comment:
Thanks for the reminder to add `platforms`. I used `platforms.linux`, because these programs aren't aarch64-specific. I tested it with `boot.binfmt.emulatedSystems = [ "riscv64-linux" ];` and `nix-build $NIXPKGS -A pkgsCross.riscv64.asahi-nvram`, and the resulting binary works just fine.

Reply:
Well, I guess you could, hah 😜